src/main/java/org/commcare/formplayer/repo/impl/PostgresFormSessionRepo.java

```diff
@@ -54,7 +54,11 @@ public void purgeFormSessions() {
             String deleteQuery = replaceTableName(
                     "delete from %s where custom_safe_cast(dateopened, '2011-01-01'::timestamp) < NOW() - INTERVAL '7 days';"
             );
-            this.jdbcTemplate.execute(deleteQuery);
+            log.info("Beginning state form session purge");
+            long start = System.currentTimeMillis();
+            int deletedRows = this.jdbcTemplate.update(deleteQuery);
```

Comment:
Does `this.jdbcTemplate.update` mean execute and return, or is there some other difference?

Reply:
`update` means "Execute the statement and expect an integer response," in jdbc, more or less. There's no specific wrappers for delete methods.